cmd/kubectl-gadget/utils/flags.go

```diff
@@ -131,7 +132,8 @@ func AddCommonFlags(command *cobra.Command, params *CommonFlags) {
 		}
 
 		// Output Mode
-		if err := params.ParseOutputConfig(); err != nil {
+		err := params.ParseOutputConfig()
+		if err != nil && command.Name() != "process" && !errors.Is(err, errors.New("\"tree\" output mode is not supported")) {
```

Comment:
IMO, it is too specific for one gadget to have it here, and it is repeated for kubectl-gadget and local-gadget. I tried this https://github.com/inspektor-gadget/inspektor-gadget/commit/a69bc1fe60fb7c571283ff64ff3d3afcc407b16a to make it more generic. WDYT?